src/utils.js

```diff
@@ -122,3 +122,15 @@ export function isActive(uri, path, exact) {
 
   return cache[[uri, path, exact]];
 }
+
+export function isFunction(object) {
+  return Object.prototype.toString.call(object).includes('Function');
```

Comment:
Same here, not sure if `typeof object === 'function'` does the same.

Reply:
Done.